src/components/admin/article/ArticleController.jsx

```diff
@@ -210,6 +203,7 @@ class ArticleController extends FalcorController {
     const pathname = `/articles/page/${page}`;
 
     const location = { pathname, state: { refresh: this.state.refresh } };
+    this.safeSetState({ refresh: false });
```

Comment:
@Helw150 I added this as I believe it was just a simple oversight from you? Right? Or is it because you were assuming when you do this the component gets unmounted so the state would reset anyway?

Reply:
Doesn't the Article Controller path get unmounted when you navigate away from the page?

Reply:
Either way adding this does no harm, and I am assuming since you added it you know something about React Router navigation that I don't! 😁

Reply:
It's mostly just being on the safe side... I feel like I remember there were times when I thought it would unmount and remount, but then it stayed mounted and just got called with `componentWillReceiveProps`, so just staying on the safe side

Reply:
Yup, no harm done :)